Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -126,6 +126,21 @@ public Element build(final CallbackProperties properties) throws SAMLAssertionBu
             final Subject subject = createSubject(properties, certificate, publicKey);
             assertion.setSubject(subject);
 
+            // add conditions statement
+            DateTime beginValidTime = new DateTime();
```

Comment:
Move this initialization into the if block.  We only need them and create the assertion condition if we detect true value for this flag